country_assignment.rb

```diff
@@ -0,0 +1,73 @@
+
+module CountryModule
+    def check_loan()
```

Comment:
Method name can be more meaningful like "is_eligible_for_loan?" OR "check_eligible_for_loan"